lib/workers/repository/update/pr/index.ts

```diff
@@ -74,6 +74,20 @@ export function updatePrDebugData(
   };
 }
 
+function hasNotIgnoredReviewers(pr: Pr, config: BranchConfig): boolean {
+  if (
+    is.nonEmptyArray(config.ignoreReviewers) &&
+    is.nonEmptyArray(pr.reviewers)
+  ) {
+    return (
+      pr.reviewers.filter(
+        (reviewer) => !config.ignoreReviewers!.includes(reviewer)
```

Comment:
```suggestion
        (reviewer) => !config.ignoreReviewers.includes(reviewer)
```

are you sure we need the `!` ?

Reply:
yes, type guard is not sufficient here 
<img width="930" alt="image" src="https://user-images.githubusercontent.com/25525736/214228815-a887563f-ff99-4513-98a5-e5bd5bd2b643.png">